test/evergreen.yml

```diff
@@ -1638,6 +1662,105 @@ tasks:
               cp -rf WT_TEST WT_TEST_$file
             done
 
+  - name: check-test-long-1
+    tags: ["check-long-test"]
+    commands: 
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: ac_cv_func_ftruncate=no
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+            ${test_env_vars|} $(pwd)/../test/csuite/random_abort/smoke.sh 2>&1
+            ${test_env_vars|} $(pwd)/../test/csuite/timestamp_abort/smoke.sh 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/test_truncated_log 2>&1
+
+  - name: check-test-long-2
+    tags: ["check-long-test"]
+    commands:
+      - func: "get project"
+      - func: "configure wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-g -Werror"
+          posix_configure_flags: --enable-silent-rules --enable-diagnostic --disable-static
+      - func: "make wiredtiger"
+        vars:
+          make_command: make all
```

Comment:
Not necessary as `make` equal `make all` in wiredtiger. Also the `${smp_command|}` in the function will be missed if setting this way.

Reply:
I see - will change it to `make`. FYI, I believe changing the make command doesn't affect `${smp_command|}` parameter.

Reply:
Oh, you are right.